File: C#/_0001multiplesof3or5.cs
/* 
 * .NET VERSION
 * _0001multiplesof3or5.cs
 * 
 * Copyright 2025 @GitHub JoergEm
 * 
 * This program is WITHOUT ANY WARRANTY; without even the implied
 * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 * 
 * 
 */
using System;

class _0001multiplesof3or5
{
	
    static void Main()
    {
		int summe = 0;
        for (int nummer = 0; nummer < 1000; nummer++) {
            if (nummer % 3 == 0 || nummer % 5 == 0) {
                summe += nummer;
            }
        }
        Console.WriteLine(summe);
	}
}

File: Erlang/0001multiples_of_3_or_5.erl
%%
%% erlang version "26"
%% 0001multiples_of_3_or_5.erl
%% Copyright 2025 @GitHub JoergEm
%%
%% This program is WITHOUT ANY WARRANTY; without even the implied
%% warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
%%


multiples_of_3_or_5(M, Limit) ->
    Nummer = (Limit - 1) div M,
    M * Nummer * (Nummer + 1) div 2.

main(_) ->
    Summe = multiples_of_3_or_5(3, 1000) + multiples_of_3_or_5(5, 1000) - multiples_of_3_or_5(15, 1000),
    io:format("Die Summe ist: ~p~n", [Summe]).